scenarios/turbine-heat-sensor/src/main/java/io/pravega/turbineheatsensor/TurbineHeatSensor.java

```diff
@@ -142,6 +142,7 @@ public static void main(String[] args) throws Exception {
         if ( !onlyWrite ) {
             consumeStats.printTotal();
         }
+        clientFactory.close();
 //        ZipKinTracer.getTracer().close();
```

Comment:
Remove this commented line in a new PR, please.

Reply:
Ok, I will do that.